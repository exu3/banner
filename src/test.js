import React from 'react'
import Banner from './'
import renderer from 'react-test-renderer'

describe('Banner', () => {
  it('is valid', () => {
    expect(Banner).toBeTruthy()
    const component = renderer.create(<Banner year={2019} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
