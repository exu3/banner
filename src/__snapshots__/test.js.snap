// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner is valid 1`] = `
<a
  href="https://hackclub.com/"
  target="_blank"
>
  <img
    alt="Hack Club"
    src="https://hackclub.com/banners/2019.svg"
    style={
      Object {
        "border": 0,
        "left": 0,
        "position": "absolute",
        "top": "10px",
        "width": "230px",
        "zIndex": "999",
      }
    }
  />
</a>
`;
